src/index.js

```diff
@@ -135,9 +140,9 @@ export default class LazilyRender extends React.Component<LazilyRenderProps, Laz
   }
 
   render() {
-    const {className, component} = this.props;
+    const {className, component, scrollContainer} = this.props;
     return React.createElement(component || 'div', {
-      ref: this.handleMount,
+      ref: element => this.handleMount(element, scrollContainer),
```

Comment:
I think it's better to move this into a class method instead of having a arrow function to avoid re renders

Reply:
Agreed, missed this one. Will update accordingly.